docs(introduction): add telemetry page (#677)
camunda/camunda-docs-manual: content/user-guide/logging.md

    <td><code>org.camunda.bpm.engine.security</code></td>
    <td>logs exceptions that occur during password hashing</td>
  </tr>
  <tr>
    <td><code>org.camunda.bpm.engine.telemetry</code></td>

>     <td>logs details for data that is sent and the problems with telemetry configuration</td>

Reviewer: ```suggestion
    <td>logs details about the sent information and error details in case the data couldn't be collected or sent</td>
```

The snippet contains the start and of `RemoveExecutionVariablesCmd`, the flush summary of the operation, and the database statements of the variable instance.

#### Telemetry Data

> In order to inspect what data is sent when [Telemetry]({{< ref "/introduction/telemetry.md" >}}) is enabled, please enable the Telemetry logger. The logger will help for troubleshooting if no data is sent.

Reviewer: ```suggestion
In order to inspect what data is sent when [Telemetry]({{< ref "/introduction/telemetry.md" >}}) is enabled, please enable the Telemetry logger. The logger will help with troubleshooting if no data is sent.
```

* `org.camunda.bpm.engine.telemetry`

> Find below an example of the log entries when the data is sent.

Reviewer: ```suggestion
Please find below an example of the log entries when the data is sent:
```
